bridge-adaptivity/docker-compose_local.yml

```diff
@@ -14,11 +14,11 @@ services:
     build:
       context: .
       dockerfile: Dockerfile_local
-    command: bash -c "sleep 5 && python manage.py runserver 0.0.0.0:8000"
+    command: bash -c "sleep 5 && python manage.py runserver 0.0.0.0:8888"
     volumes:
       - .:/bridge-adaptivity
     ports:
-      - "8000:8000"
+      - "8888:8888"
```

Comment:
Sorry - Can we change the port, or at least the host port being mapped to, to something besides 8888? (8888 is the default for jupyter notebooks)